portfolio/src/main/webapp/script.js

```diff
@@ -104,10 +105,83 @@ function loadPage() {
     loadLanguageDropdown();
 }
 
```

Comment:
Try to maintain a consistent style, you appear to have added an extra two lines of vertical whitespace here.

There's a collection of Style Guides which provide the "Google Standards" for consistent style https://google.github.io/styleguide

On the topic of vertical whitespace in JavaScript:
https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace